webapp/plugins/local/footer-address/index.js

```diff
@@ -26,7 +26,11 @@ export const decorateFooter = (Footer, { React, PropTypes }) => {
     render() {
       const { wallets, footerWidgets = [] } = this.props;
 
-      const address = wallets.primary ? wallets.primary.receiveAddress : '...';
+      const address = wallets.accountInfo
```

Comment:
I think that a nested ternary operator is hard to read even tho having a single expression is nice

Reply:
Yeah, don't disagree. So you think something like `if (wallets.accountInfo && wallets.accountInfo.primary)` followed by assignment and an else would be better?